database/errors.go

```diff
@@ -6,7 +6,8 @@ import (
 
 var (
 	// ErrTableNotFound is returned when the targeted table doesn't exist.
-	ErrTableNotFound = errors.New("table not found")
+	// The table name is added to the error.
+	ErrTableNotFound = errors.New("Error: no such table:")
```

Comment:
There is no need to change this. See [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

```suggestion
	ErrTableNotFound = errors.New("table not found")
```